cmd/kube-proxy/app/server_others.go

```diff
@@ -110,6 +114,26 @@ func newProxyServer(
 		return nil, err
 	}
 
+	// create client for each tenant partition, if applicable
+	var clients []clientset.Interface
+	if len(config.TenantPartitionKubeConfig) == 0 {
+		klog.V(3).Infof("TenantPartitionKubeConfig not set. fall back to scale-up cluster.")
+		clients = []clientset.Interface{client}
+	} else {
+		klog.V(3).Infof("scale-out cluster; make KubeClients based on TenantPartitionKubeConfig arg: %v", config.TenantPartitionKubeConfig)
+		kubeConfigFiles, existed := genutils.ParseKubeConfigFiles(config.TenantPartitionKubeConfig)
+		if !existed {
+			klog.Fatalf("Kubeconfig file(s) [%s] for tenant server does not exist", config.TenantPartitionKubeConfig)
+		}
+		clients = make([]clientset.Interface, len(kubeConfigFiles))
+		for i, kubeConfigFile := range kubeConfigFiles {
+			clients[i], err = clientutil.CreateClientFromKubeconfigFile(kubeConfigFile, "kubelet")
+			if err != nil {
+				klog.Fatalf("failed to initialize kubelet client from kubeconfig [%s]: %v", kubeConfigFile, err)
```

Comment:
here too. should be kube-proxy instead of kubelet

Reply:
will modify to kube-proxy